docs/Training-SAC.md

```diff
@@ -106,17 +105,16 @@ there may not be any new interesting information between steps, and `train_inter
 
 Typical Range: `1` - `5`
 
-### Number of Updates
+### Steps Per Update
 
-`num_update` corresponds to the number of mini batches sampled and used for training during each
-training event. In SAC, a single "update" corresponds to grabbing a batch of size `batch_size` from the experience
-replay buffer, and using this mini batch to update the models. Typically, this can be left at 1.
+`steps_per_update` corresponds to the number agent steps (actions) taken for each mini-batch sampled and used during training. In SAC, a single "update" corresponds to grabbing a batch of size `batch_size` from the experience
+replay buffer, and using this mini batch to update the models. Typically, this should be greater than 1.
 However, to imitate the training procedure in certain papers (e.g.
 [Kostrikov et. al](http://arxiv.org/abs/1809.02925), [Blondé et. al](http://arxiv.org/abs/1809.02064)),
 we may want to update N times with different mini batches before grabbing additional samples.
-We can change `train_interval` and `num_update` to N to accomplish this.
+We can change `steps_per_update` to lower than 1 to accomplish this.
 
-Typical Range: `1`
+Typical Range: `10` - `20`
```

Comment:
What's the reasoning behind this range?  It's not atypical to perform an update on every step of an agent.  Granted, that doesn't scale if there are ten agents and every single one takes a step per timestep in which case the minimum is 10.  Maybe this should be written as a function of the number of agents/envs?

Reply:
This was to keep in-line with our previous SAC behavior, which was one update per env step (~10 agent steps). From empirical trials I haven't noticed much of a sample efficiency improvement pushing this down to 1 - in fact it caused a huge wallclock time degradation. This was even with Snoopy Pop, the slowest env we have. 

This number should be independent of # of agents/envs unlike `num_updates`. Maybe we should lower the range to `1-20`?